compose/material3/material3/src/androidMain/kotlin/androidx/compose/material3/PrecisionPointer.android.kt

```diff
@@ -26,31 +26,26 @@ import androidx.compose.runtime.DisposableEffect
 import androidx.compose.runtime.Immutable
 import androidx.compose.runtime.MutableState
 import androidx.compose.runtime.State
-import androidx.compose.runtime.compositionLocalOf
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
+import androidx.compose.runtime.staticCompositionLocalOf
 import androidx.compose.ui.platform.LocalContext
 import androidx.core.content.getSystemService
 
-internal actual val shouldUsePrecisionPointerComponentSizing: Boolean
```

Comment:
I didn't find an issue in YouTrack for implementing this (we had to add it during the previous merges).

Please, create "Implement `shouldUsePrecisionPointerComponentSizing` setting" "Introduced in ... merge" (now it is not an expect/actual, but we still need to set it)

Reply:
There is this - https://youtrack.jetbrains.com/issue/CMP-9305 , I added a comment there